proxlist/proxies.py

```diff
@@ -30,3 +30,11 @@ def random_proxy() -> str:
 
 def list_proxies() -> List[str]:
     return PROXY_LIST
+
+
+def proxy_dict() -> dict:
```

Comment:
```suggestion
def proxy_dict() -> Dict[str, str]:
```

Reply:
For Python to be happy with type hints on Python prior to 3.9, you'll want to add `from typing import Dict` to the top.